GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerFixture/DehydrateTests.cs

```diff
@@ -198,6 +567,11 @@ private ProcessResult RunDehydrateProcess(bool confirm, bool noStatus)
                 dehydrateFlags += " --no-status ";
             }
 
+            if (foldersToDehydrate.Length > 0)
+            {
+                dehydrateFlags += $" --folders {string.Join(";", foldersToDehydrate)}";

```

Comment:
Should path separator be a const somewhere (that both client / server ends of pipe can read)?